# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccessibilityService.cpp
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/base/nsAccessibilityService.cpp
--- a/accessible/base/nsAccessibilityService.cpp	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/base/nsAccessibilityService.cpp	Wed Nov 29 03:53:20 2023 +0000
@@ -1006,7 +1006,7 @@
 
 void nsAccessibilityService::GetStringEventType(uint32_t aEventType,
                                                 nsAString& aString) {
-  NS_ASSERTION(
+  static_assert(
       nsIAccessibleEvent::EVENT_LAST_ENTRY == ArrayLength(kEventTypeNames),
       "nsIAccessibleEvent constants are out of sync to kEventTypeNames");
 